### ISSUE TITLE
support kitty's protocol

### ISSUE THREAD

**Issue by teto** [wookayin/python-imgcat#4]
wow that looks dope, would it be possible to support kitty (a cross platform python terminal) as well ? it seems to have its own format too https://sw.kovidgoyal.net/kitty/graphics-protocol.html (the link provides a python function to use the protocol).

Another implementation reference could be:
https://pypi.org/project/pixcat/#description

**Comment by wookayin** [comment]
Yes, I plan to support kitty and sixel protocols so that it can work with another terminal emulators than iTerm2. Stay tuned!

**Comment by wookayin** [comment]
tmux is **very stubborn** in supporting image display (either in iTerm2 or kitty). Unfortunately, there is no way to display images inside tmux. See: https://github.com/kovidgoyal/kitty/issues/413

**Comment by wookayin** [comment]
Using a very complicated [tmux passthrough sequence](https://github.com/tmux/tmux/issues/846), I managed to make images rendered on kitty, but there is some bugs on clearing up graphics. See [kitty#2457](https://github.com/kovidgoyal/kitty/issues/2457) for details.

**Comment by teto** [comment]
I dont use tmux much, would that work better without tmux on a plain kitty ?

**Comment by wookayin** [comment]
@teto Yes I agree, will push an early version (without or with tmux) soon.

**Comment by davvid** [comment]
tmux has a passthrough option these days. It makes images display fine inside of tmux inside kitty:

```
set -gq allow-passthrough on
```
